### ISSUE TITLE
I'm listed on this site, but i'm not on the list it's claiming i'm on

### ISSUE THREAD

**Issue by quat1024** [misea/find-masto-folks#6]
My account appears on this page: https://find.sciences.social/search/accounts/@quat@woof.group?field=history-and-philosophy-of-science . I found out because this mirror of my account is indexed by Google (and I requested to disable indexing by search engines on my mastodon account.)

The page says "To add or remove yourself from the list or to modify your information, see the [list page](https://oliviodare.github.io/mastodon-hps/)".

1. I can't message the list owner because `scholar.social` instance-blocked us.
2. I am not an academic in this field.
3. If I go to that list page and Ctrl-F for my name, my account is not even contained on the list?

So i'm not sure what's happening but I would prefer if my account was not listed here?

**Comment by quat1024** [comment]
Looks like you can just change the URL to whatever account you want, and this site will display the account even if the account isn't on the list.

**Comment by misea** [comment]
Yes - that looks to be true. It will call the public API for any account that is passed on the URL. Should close that loophole. It doesn't expose private info, but it would be a way for people to maybe pretend they were on a list. Thanks

**Comment by misea** [comment]
OK this should be fixed now. Note that there was never any leakage of information that wasn't available on the internet. Google was somehow putting in handles that it found elsewhere. I updated robots.txt so that google and all other crawlers, should not index anymore, but there are no guarantees they will follow the rules.
